modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2934,12 +2934,47 @@ definition {
 	}
 
 	@description = "LPS-139803 - Verify it the Object Entry Title of a BigDecimal type is displayed on the Relationship field when adding an entry"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanViewObjectEntryTitleBigDecimalOnRelationshipField {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "BigDecimal",
+			fieldName = "customObjectField",
+			fieldType = "BigDecimal",
+			isRequired = "false",
+			objectName = "CustomObject");
 
-		// TODO LPS-141611 CanViewObjectEntryTitleBigDecimalOnRelationshipField pending implementation
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObject",
+			objectName_2 = "CustomObject",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+		CreateObject.selectTitleField(fieldLabel = "BigDecimal");
+
+		Button.clickSave();
+
+		ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+			fieldName = "customObjectField",
+			objectName = "CustomObject",
+			value = "123.123456");
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
 
+		ObjectPortlet.addSingleFieldEntryViaUI(
+			entry = "987.987654",
+			entryOption = "123.123456");
```

Comment:
You don't need to add an entry for this test. You only need to click on the Relationship field and assert the entry value. You can use the `CreateObject.assertAddedEntryRelationshipIsDisplayed` macro for this. See example on the `CanViewObjectEntryTitleStringOnRelationshipField` test.

Reply:
Reviewed! I thought other way. It was more simple.

Thanks!